### ISSUE TITLE
Document how to resolve `Null key` error in `UPSERT` sources

### ISSUE THREAD

**Issue by guswynn** [MaterializeInc/materialize#18953]
### Documentation request

This is a followup task for incident 48.

The "null key in upsert source" error in the incident (`ERROR:  Envelope error: Upsert: Null key: record with NULL key in UPSERT source in partition 0`) should link to a troubleshooting guide on how to resolve it. Note that the error gives a better hint now (https://github.com/MaterializeInc/materialize/pull/18887), but that it does NOT give specific advice on how to generate the specific record that unblocks the source.

Kafka clients are really inconsistent, so the advice in the documentation should recommend using the `kcat` command, and offer a specific command users can paste their credentials into.

### Affected Pages

https://materialize.com/docs/ops/troubleshooting/

### Related work

https://github.com/MaterializeInc/materialize/issues/17826

**Comment by guswynn** [comment]
omg sorry about the duplicate marta!!

**Comment by morsapaes** [comment]
It's the intention that counts, @guswynn! ❤️
